### PR TITLE
catch anchor in event bubbling

### LINKED ISSUE #9: Capturing all anchor clicks
> Use JavaSript to capture all anchor clicks, and `console.log()` the element which has been clicked.

You might take it too complicated. Please use one line to achieve this.

### PR DESCRIPTION
**Pull request by trust2065**
fix #9 

Please find anchors show in below images

image
image

and find console log in dev tool.

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,17 +41,15 @@
...
+              <h1>
+                <a
+                  class="textBox--no-decoration textBox__head textBox__head--grey textBox__head--upper"
+                  href="http://0rz.tw/AaW9E"
+                  target="_blank"
                 >
                   Answer your body's needs
-                </h1>
-              </a>
+                </a>
```
Comment (josephj) [inline index.html]: What's the purpose of having a link here? Is this part of the spec? Does this benefits to the user?

Comment (trust2065) [reply index.html]: It doesn't have to be a link.

**Comment by trust2065** [comment]
-- I'm talking to myself 
It's a bit mess when many PR doing at the same time for me.
But I think no matter what each PR should able to work alone, and have the right to merge back.
Since it might be other developer working on each PR, I should pretend no knowing things in other branch.

**Comment by josephj** [comment]
One issue one PR. Merge it back fast. While it's not enough, create another issue. Don't be greedy.

**Comment by josephj** [comment]
Also triage first for prioritisation and tickets merging

**Comment by trust2065** [comment]
Don't really know what you mean 😕 

> Also triage first for prioritisation and tickets merging

**Comment by josephj** [comment]
基本上就是把票給 管理 好
有票進來時就先 Triage 分類
GitHub 有 Label 功能
如果有重複的就先關掉
讓票成為你的施工順序及藍圖

**Comment by josephj** [comment]
對啊 有好幾個病人來看病 你為啥要一起看咧 
或者是一個病人有很多問題 你為啥要一次解決內科 感冒 皮膚病 還開刀哩

**Comment by josephj** [comment]
我看你的 pull request 時也覺得怎麼都沒完沒了
後來才發現你想要一個 pr 做很多事

**Pull request merged**